graphs/possible_bipartition.py

```diff
@@ -5,8 +5,33 @@ def possible_bipartition(dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(N+E) N-number of nodes, E -number of edges
+        Space Complexity: O(N)??
```

Comment:
🪐 Yes! Space complexity will be O(N) because the data structures you create, `queue`, `visited`, `group1`, and `group2` can each hold at most N nodes (aka dogs).